packages/salesforcedx-test-utils-vscode/src/testrunner.ts

```diff
@@ -41,21 +41,23 @@ function configure(mochaOpts: any): void {
     // This will fix the pathing for windows. This behavior is not seen in the system-tests, appears to be only when we use vscode's test bin to run tests
     if (process.platform === 'win32') {
       xmlPath = paths.normalize(paths.join(process.cwd(), '..', '..'));
+    } else {
+      xmlPath = paths.join(__dirname, '..', '..', '..', '..');
```

Comment:
Its failing currently without this on an EACCES permissions error.

Reply:
@nrkruk You know what path this should really resolve to? I used the root of the project, but I don't know if it should be the root of each package.